### PR TITLE
Report breakdown of vale errors per module

### PR DESCRIPTION
**Pull request by themr0c**
Report breakdown of vale errors per module

* Script in the tools directory
* Workflow on push using this script

Current output:

```
./tools/count_vale_errors.sh                                                                                                                                                                             
vale version 2.10.2
# Breakdown of vale errors per module:
modules/administration-guide: 0
modules/contributor-guide: 11
modules/end-user-guide: 0
modules/extensions: 40
modules/glossary: 0
modules/hosted-che: 8
modules/installation-guide: 193
modules/overview: 0
```

### CODE REVIEW

File: tools/count_vale_errors.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```
Comment (benoitf) [inline tools/count_vale_errors.sh:1]: you should add license on top of the file

File: tools/count_vale_errors.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+vale -v 
+echo "# Breakdown of vale errors per module:"
+
+for module in modules/*
+    do
+    echo -n "$module: "
```
Comment (benoitf) [inline tools/count_vale_errors.sh]: it doesn't pass shellcheck, so probably use bash instead of sh

**Review by benoitf — APPROVED** [review]

File: tools/count_vale_errors.sh (hunk 2 above)
Comment (themr0c) [reply tools/count_vale_errors.sh]: Oh that's strange, shell check was passing locally. I'll fix it after some PTO.

**Comment by themr0c** [comment]
The vale report on the yaml is unexpected. I'll have to fix this also.

**Comment by boczkowska** [comment]
@themr0c we need to disable vale for this file.

File: tools/count_vale_errors.sh (hunk 1 above)
Comment (themr0c) [reply tools/count_vale_errors.sh:1]: Done.

File: tools/count_vale_errors.sh (hunk 2 above)
Comment (themr0c) [reply tools/count_vale_errors.sh]: Indeed. Last minute change introduced the error. Nicely spotted. Fixed.

**Review by yhontyk — APPROVED** [review]

**Pull request merged**